### DISCUSSION TITLE
Libghostty File Manager?

### DISCUSSION THREAD

**Discussion by DeadlyDad** [ghostty-org/ghostty#3623] [Feature Requests, Ideas] [answered]
If anyone is bored and looking for something to do, a libghostty build of NDN (http://ndn.muxe.com) would be incredible.  (If you are unfamiliar with it, it is essentially what you would get if a crack addict spent the last thirty years tweaking Norton Commander.  Almost every single key has at least two or three functions, and the text/code editor directly supports regex.)

It would *NOT* be a trivial task by any means, but it would *DEFINITELY* be a worthy one.

**Comment by mitchellh** [comment, marked as answer]
libghostty isn't yet available. But cool idea when it is. 😄

**Reply by MatthiasGrandl** [reply]
Is libghostty going to be a general purpose GUI library?

**Reply by jcollie** [reply]
No, it will not.
